client/app/match/match.html

```diff
@@ -1,6 +1,6 @@
 <!-- This is the page that is shown when we are trying to match with the other people in our session -->
 <!-- We should be shown a movie, and then choose 'yes' or 'no' -->
-<div id="match" class="container">
+<div id="match" class="container" ng-controller='MatchController'>
```

Comment:
We're already specifying the match controller in app.js right? Don't need to include here otherwise it'll get called twice.